sql/snowflake/SnowflakeLexer.g4

```diff
@@ -302,6 +302,7 @@ ERROR_ON_NONDETERMINISTIC_MERGE:                       'ERROR_ON_NONDETERMINISTI
 ERROR_ON_NONDETERMINISTIC_UPDATE:                      'ERROR_ON_NONDETERMINISTIC_UPDATE';
 ESCAPE:                                                'ESCAPE';
 ESCAPE_UNENCLOSED_FIELD:                               'ESCAPE_UNENCLOSED_FIELD';
+EVENT:                                                 'EVENT';
```

Comment:
Please add `EVENT` to the [non_reserved_words](https://github.com/antlr/grammars-v4/blob/master/sql/snowflake/SnowflakeParser.g4#L3445) list. I suppose it can be identifier.